pkg/sql/compile/scope.go

```diff
@@ -692,10 +692,11 @@ func newParallelScope(s *Scope, ss []*Scope) (*Scope, error) {
 					Idx:     in.Idx,
 					IsFirst: in.IsFirst,
 					Arg: &group.Argument{
-						Aggs:      arg.Aggs,
-						Exprs:     arg.Exprs,
-						Types:     arg.Types,
-						MultiAggs: arg.MultiAggs,
+						Aggs:           arg.Aggs,
+						Exprs:          arg.Exprs,
+						Types:          arg.Types,
+						MultiAggs:      arg.MultiAggs,
```

Comment:
How does this cross nodes?

Reply:
It will be transferred only between tablescannode and aggnode, stored in Scopes when compiletablescan and put into group.argument.